content/python/concepts/file-methods/terms/remove/remove.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.remove()'
+Description: 'Allows the user to delete a file.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Methods'
+  - 'Files'
+  - 'Functions'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The Python `.remove()` file method allows the user to delete a file if it exists in the system or computer. This method is provided by the `os` module which must be imported beforehand.
+
+**Note** A `FileNotFoundError` is raised if the file is not found or isn't in the current directory.
```

Comment:
```suggestion
**Note**: A `FileNotFoundError` is raised if the file is not found or isn't in the current directory.
```

Reply:
@KTom101 maybe we can have an entry that includes the `FileNotFoundError` 🤔 Do you think it would be a good idea to make an issue requesting this?

Reply:
@yangc95 Yes that would be a good idea. There actually other Errors I wanted to include IsADirectory Error and the OSError which are errors you can get using these methods. but I felt it would be too long here.